docs/guides/node-operator-manual.md

```diff
@@ -149,13 +169,29 @@ fields except `pubkey` and `signature` from the array items.
 Never share the generated mnemonic and your private keys with anyone, including the protocol members
 and DAO holders.
 
+[`eth2.0-deposit-cli`]: https://github.com/ethereum/eth2.0-deposit-cli/releases
 [a fork of `eth2.0-deposit-cli`]: https://github.com/lidofinance/eth2.0-deposit-cli
 [`lidofinance/deposit-cli`]: https://hub.docker.com/repository/docker/lidofinance/deposit-cli
 
+### Validating the keys
+
+It's vital to check the correctness of the keys before submitting them.
+
+If you will be submitting keys using Lido's submitter, they will be checked before submitting - the "submit" button won't be active unless keys are validated and they are valid.
+
+If you will be submitting keys manually via Lido contract, you can use Lido CLI, it's as simple as having Python with pip installed and running:
```

Comment:
If you will be submitting keys manually via Lido contract, you can use Lido CLI. 
It's distributed as a pip package: